### PR TITLE
[Doc] add known limitation and settings for connection issue

### PR DESCRIPTION
**Pull request by kaisecheng**
Fixed: #166

Added a section for known limitation to document the Sybase issue.

### CODE REVIEW

File: docs/input-jdbc.asciidoc [hunk 1]
```diff
@@ -176,6 +176,29 @@ input {
 }
 ---------------------------------------------------------------------------------------------------
 
+==== Known limitations
+
+Theoretically, this plugin can function with any JDBC driver.
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:181]:
```suggestion
The JDBC input plugin leverages the "sequel" library to work with databases through their JDBC drivers.
```

File: docs/input-jdbc.asciidoc [hunk 2]
```diff
@@ -176,6 +176,29 @@ input {
...
+However, the implementation of drivers may vary, potentially leading to unexpected behavior in connection pool management.
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:182]:
```suggestion
However, the implementation of drivers may vary, potentially leading to unexpected behavior.
```

File: docs/input-jdbc.asciidoc [hunk 3]
```diff
@@ -176,6 +176,29 @@ input {
...
+
+===== Unable to reuse connection
+
+Stale connections may return to the pool, causing the plugin reuse closed connections. Sybase is affected by this issue.
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:186]: Some databases - such as Sybase or SQL Anywhere - may have issues with stale connections, timing out between scheduled runs and never reconnecting.

File: docs/input-jdbc.asciidoc [hunk 4]
```diff
@@ -176,6 +176,29 @@ input {
...
+
+One solution is to enable <<plugins-{type}s-{plugin}-jdbc_validate_connection>> and set <<plugins-{type}s-{plugin}-jdbc_validation_timeout>> to a shorter interval than the <<plugins-{type}s-{plugin}-schedule>>.
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:188]:
```suggestion
To ensure connections are valid before queries are executed, enable <<plugins-{type}s-{plugin}-jdbc_validate_connection>> and set <<plugins-{type}s-{plugin}-jdbc_validation_timeout>> to a shorter interval than the <<plugins-{type}s-{plugin}-schedule>>.
```

File: docs/input-jdbc.asciidoc [hunk 5]
```diff
@@ -176,6 +176,29 @@ input {
...
+
+[source,ruby]
+---------------------------------------------------------------------------------------------------
+input {
+  jdbc {
+    schedule => "* * * * *"       # run every minute
+    jdbc_validate_connection => true
+    jdbc_validation_timeout => 55 # 55 seconds
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:196]:
```suggestion
    jdbc_validation_timeout => 50 # 50 seconds
```

File: docs/input-jdbc.asciidoc [hunk 6]
```diff
@@ -176,6 +176,29 @@ input {
 }
 ---------------------------------------------------------------------------------------------------
 
+==== Database-specific considerations
+
+The JDBC input plugin leverages the "sequel" library to query databases through their JDBC drivers.
+The implementation of drivers will vary, however, potentially leading to unexpected behavior.
+
+===== Unable to reuse connection
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:184]:
```suggestion
===== Unable to reuse connections
```

File: docs/input-jdbc.asciidoc [hunk 7]
```diff
@@ -176,6 +176,29 @@ input {
...
+
+Stale connections may return to the pool, causing the plugin reuse closed connections. Sybase is affected by this issue.
```
Comment (jsvd) [inline docs/input-jdbc.asciidoc:186]:
```suggestion

Some databases - such as Sybase or SQL Anywhere - may have issues with stale connections, timing out between scheduled runs and never reconnecting.
```

**Review by roaksoax — APPROVED** [review]
lgtm!

**Review by jsvd — APPROVED** [review]
LGTM

**Pull request merged**